src/main/rx/fport.c

```diff
@@ -184,7 +189,11 @@ static void fportDataReceive(uint16_t c)
         } else {
             clearToSend = false;
         }
+
         telemetryFrame = false;
+    } else if (skipUntilStart) {
+        frameStarting = false;
+        bufferPosition = 0;
```

Comment:
both `bufferPosition` may be reset once when setting `skipUntilStart`

Reply:
I got an idea for that.
  ￼  ￼￼